prometheus/src/main/java/org/opensearch/sql/prometheus/storage/PrometheusMetricTable.java

```diff
@@ -97,10 +97,6 @@ public Map<String, ExprType> getFieldTypes() {
   public PhysicalPlan implement(LogicalPlan plan) {
     PrometheusMetricScan metricScan =
         new PrometheusMetricScan(prometheusClient);
-    if (prometheusQueryRequest != null) {
-      metricScan.setRequest(prometheusQueryRequest);
```

Comment:
I see `request` field is still in use in metric scan. So where we set it now?

Reply:
It is set in https://github.com/opensearch-project/sql/blob/main/prometheus/src/main/java/org/opensearch/sql/prometheus/storage/implementor/PrometheusDefaultImplementor.java#L109 here

Implementation of query_range and ppl.prometheus_http_requests_total got coupled with weird logic. Need to refactor in another PR.

Reply:
query_range...we directly set the prometheusQueryRequest.
In case of `source = my_prometheus.prometheus_http_requests_total` we set these parameters in PrometheusDefaultImplementator class.